### DISCUSSION TITLE
How to change red accent on search bar, pins and recents?

### DISCUSSION THREAD

**Discussion by harukayamazaki** [Godiesc/firefox-gx#77] [Q&A]
I have been able to change the default color using colors but other elements are still red. Could someone direct me to the lines of css I need to alter to change the accents and the highlight color when selecting anything from pins and recents.

**Comment by Godiesc** [comment]
in the main readme there are information for change a lot of thing, for the accent color in new tab page, addons page and preferences page you have this in `ogx_root-personal.css`:

![172070426-0de79289-eaa3-4826-ac62-af6230cdf877](https://github.com/Godiesc/firefox-gx/assets/22057609/4ee2c310-1b40-481d-97f6-d00cc85e3420)

change to the color you want.
